src/System.Private.DataContractSerialization/src/System/Xml/XmlNodeWriter.cs

```diff
@@ -10,25 +10,41 @@
 using System.Globalization;
 using System.Runtime.Serialization;
 using System.Collections.Generic;
-
+using System.Threading.Tasks;
 
 namespace System.Xml
 {
     internal abstract class XmlNodeWriter
     {
         public abstract void Flush();
+        public virtual Task FlushAsync()
```

Comment:
Both XmlBinaryNodeWriter and XmlUTF8NodeWriter inherit from this class. We have no plan to add async operations for XmlBinaryNodeWriter for now.